sample-apps/react/react-video-demo/src/components/Header/Header.module.css

```diff
@@ -59,9 +59,13 @@
 
 .names {
   @apply hidden sm:flex mr-8 ml-8 text-video-white;
+  display: block;
   max-width: 200px;
   overflow: hidden;
   white-space: nowrap;
+
+  text-overflow: ellipsis;
+
```

Comment:
Any particular reason for the spacing between this single declaration? Just curious 🤔